### DISCUSSION TITLE
Occlusion without LiDar

### DISCUSSION THREAD

**Discussion by bildekrokodil** [google/model-viewer#3936] [General]
Hi all,

I have a iphone 12 pro with LiDar scanner and the good thing about it is than when I show a model with modelviewer and then you go activate AR you can place this model in your environment (So far nothing new). Thanks to the lidar scanner the object can also be placed behind a real surface like a chair for example.

When I do this with a phone without lidar and even any high super end Android phone. (Newest models) It will never occlude the object. It is always in front of everything. I thought this would be just a few months waiting before android can do this to but no! Still nothing. Do I need do set some extra settings to make it work with depth camara's? Do I do something wrong?

Kind regards

**Comment by elalish** [comment]
Actually it is available on Android (though it doesn't work as well), we just haven't implemented it yet. PRs welcome!
